### PR TITLE
Add the rest of the PIDs from E1.37-1

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

File: firmware/src/dimmer_model.c [hunk 1]
```diff
@@ -48,6 +51,19 @@ static const char LOCK_STATE_DESCRIPTION_SUBDEVICES_LOCKED[] =
...
+static const char CURVE_DESCRIPTION2[] = "Modified Linear";
+static const char CURVE_DESCRIPTION3[] = "Square";
+static const char CURVE_DESCRIPTION4[] = "Modified Square";
+
+static const char OUTPUT_RESPONSE_DESCRIPTION1[] = "Fast";
+static const char OUTPUT_RESPONSE_DESCRIPTION2[] = "Slow";
+
+static const char MODULATION_FREQUENCY_DESCRIPTION1[] = "50 Hz";
+static const char MODULATION_FREQUENCY_DESCRIPTION2[] = "60 Hz";
+static const char MODULATION_FREQUENCY_DESCRIPTION3[] = "1000 Hz";
```
Comment (peternewman) [inline firmware/src/dimmer_model.c]: Call it 1 kHz perhaps?

File: firmware/src/dimmer_model.c [hunk 2]
```diff
@@ -528,6 +588,181 @@ int DimmerModel_SetBurnIn(const RDMHeader *header,
...
+  *ptr++ = 8u;  // level resolution
+  *ptr++ = 1u;  // split levels supported
+  return RDMResponder_AddHeaderAndChecksum(header, ACK, ptr - g_rdm_buffer);
+}
+
+int DimmerModel_GetMinimumLevel(const RDMHeader *header,
+                                UNUSED const uint8_t *param_data) {
+  uint8_t *ptr = g_rdm_buffer + sizeof(RDMHeader);
+  ptr = PushUInt16(ptr, g_active_device->min_level_increasing);
+  ptr = PushUInt16(ptr, g_active_device->min_level_decreasing);
```
Comment (peternewman) [inline firmware/src/dimmer_model.c]: You're not handling the split level stuff here.

**Comment by peternewman** [comment]
Generally LGTM

File: firmware/src/dimmer_model.c (hunk 1 above)
Comment (nomis52) [inline firmware/src/dimmer_model.c]: Done

File: firmware/src/dimmer_model.c (hunk 2 above)
Comment (nomis52) [inline firmware/src/dimmer_model.c]: How so? this supports separate levels for increasing / decreasing.

Comment (peternewman) [inline firmware/src/dimmer_model.c]: Quite correct, I'd misread it somehow.

**Comment by peternewman** [comment]
LGTM

**Pull request merged**